.editorconfig

```diff
@@ -0,0 +1,27 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+# Tabs in JS unless otherwise specified
+[**.js]
+indent_style = tab
+indent_size = 4
+
+[**.jsx]
+indent_style = tab
+indent_size = 4
+
+[**.html]
+indent_style = tab
+indent_size = 4
+
+[**.css]
+indent_style = tab
+indent_size = 4
```

Comment:
I could hug you for adding an .editorconfig :smile:

![](http://media3.giphy.com/media/xar2IQ9iub7Xi/giphy.gif)